docs-ref-services/intune.md

```diff
@@ -24,17 +24,33 @@ Azure Intune is a cloud-based enterprise mobility management (EMM) service that
 
 Intune integrates closely with Azure Active Directory (Azure AD) for identity and access control, and Azure Information Protection for data protection.
 
-## Install the modules with npm
+## Management Package
+
+### Install npm modules
 
 Use npm to install the Azure Intune modules for Node.js
 
-### Management
 ```bash
 npm install azure-arm-intune
 ```
 
-##Example
+### Example
+
+Authenticate, create a client and retrieve a location by host name.
+
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const IntuneResourceManagementClient = require('azure-arm-intune');
+
+msRestAzure.interactiveLogin().then(credentials => {
+  const client = new IntuneResourceManagementClient(credentials);
+  client.getLocationByHostName(null).then((error, result) => {

```

Comment:
https://github.com/Azure/azure-sdk-for-node/blob/e90415ddd004e32a7ceff548a3d47d34866ec09c/lib/services/intune/lib/intuneResourceManagementClient.d.ts#L125

I think that is it is a Promise with a single result. see the link above